guppylang/compiler/stmt_compiler.py

```diff
@@ -53,10 +53,10 @@ def _unpack_assign(self, lhs: ast.expr, port: Wire, node: ast.stmt) -> None:
         """Updates the local DFG with assignments."""
         if isinstance(lhs, PlaceNode):
             self.dfg[lhs.place] = port
-        elif isinstance(lhs, ast.Tuple):
-            types = [get_type(e).to_hugr() for e in lhs.elts]
+        elif isinstance(lhs, TupleUnpack):
+            types = [get_type(e).to_hugr() for e in lhs.pattern.left]
             unpack = self.builder.add_op(ops.UnpackTuple(types), port)
-            for pat, wire in zip(lhs.elts, unpack, strict=True):
+            for pat, wire in zip(lhs.pattern.left, unpack, strict=True):
```

Comment:
This is just a temporary fix to make compilation for the existing tests work. It will be replaced with the updated Hugr lowering